src/container/cli.c

```diff
@@ -1111,11 +1114,21 @@ dc_cont_set_prop(tse_task_t *task)
 	crt_endpoint_t			 ep;
 	crt_rpc_t			*rpc;
 	struct cont_set_prop_args	 arg;
+	int				 i;
 	int				 rc;
 
 	args = dc_task_get_args(task);
 	D_ASSERTF(args != NULL, "Task Argument OPC does not match DC OPC\n");
 
+	for (i = 0; i < args->prop->dpp_nr; i++) {
+		if (args->prop->dpp_entries[i].dpe_type ==
+		    DAOS_PROP_CO_MAX_OID) {
```

Comment:
I guess it doesn't save many lines, but you could use daos_prop_entry_get to make this check a little simpler